### PR TITLE
docs: link to redefined features

### PR DESCRIPTION
**Pull request by maxteufel**
And drop redundant copy of comments of redefined features.

### CODE REVIEW

File: src/dev/flang/tools/docs/Html.java [hunk 1]
```diff
@@ -137,6 +137,38 @@ private String summary(AbstractFeature af)
       + "</div>";
   }
 
+  /**
+   * list of features that are redefined by feature af
+   * @param af
+   * @return list of redefined features, as HTML
+   */
+  private String redefines(AbstractFeature af, boolean recurse)
```
Comment (michaellilltokiwa) [inline src/dev/flang/tools/docs/Html.java:145]: I had trouble understanding the `recurse` option. If you split this up into a recursive part and one that wraps the result in the `div>ul` then it should be easier to read. I also think the `af.redefines().isEmpty()` should not be necessary?

File: src/dev/flang/tools/docs/Html.java [hunk 2]
```diff
@@ -138,6 +138,34 @@ private String summary(AbstractFeature af)
...
+    if (af.redefines().isEmpty())
+      {
+        return "";
+      }
+
+    return "<div class='fd-redefines'><br />redefines: <br /><ul>" + redefines0(af) + "</ul><br /></div>";
+  }
+
+
+  private String redefines0(AbstractFeature af)
```
Comment (fridis) [inline src/dev/flang/tools/docs/Html.java:157]: Can you add a comment, in particular explaining  —if correct— that this is a helper for `redefines` and what this does.

File: src/dev/flang/tools/docs/Html.java [hunk 3]
```diff
@@ -138,6 +138,34 @@ private String summary(AbstractFeature af)
...
+  /**
+   * list of features that are redefined by feature af
+   * @param af
+   * @return list of redefined features, as HTML
+   */
+  private String redefines(AbstractFeature af)
+  {
+    if (af.redefines().isEmpty())
+      {
+        return "";
```
Comment (fridis) [inline src/dev/flang/tools/docs/Html.java:150]: try to avoid having multiple `return` statements, makes the code harder to understand and debug.

**Review by fridis — COMMENTED** [review]
just my nitpicking...

**Comment by fridis** [comment]
@maxteufel, can you resolve the conflicts?

**Pull request merged**